fix compile probs for distcc and multicore cpu
2009 Feb 23; hd_brummy@g.o
diff -Naur xxvautotimer-0.1.2.orig/Makefile xxvautotimer-0.1.2/Makefile
--- xxvautotimer-0.1.2.orig/Makefile	2009-02-23 21:01:22.046514520 +0100
+++ xxvautotimer-0.1.2/Makefile	2009-02-23 21:02:00.720161608 +0100
@@ -107,7 +107,7 @@
 
 
 $(MYSQLWRAPPERDIR)/$(MYSQLWRAPPERLIB):
-	cd $(MYSQLWRAPPERDIR);make all
+	${MAKE} -C $(MYSQLWRAPPERDIR) all
 
 
 libvdr-$(PLUGIN).so: $(OBJS) $(MYSQLWRAPPERDIR)/$(MYSQLWRAPPERLIB)
